sourcegraph/sourcegraph
# Code Insights: Add header actions to the insight standalone page

Closes https://github.com/sourcegraph/sourcegraph/issues/34759
Based on https://github.com/sourcegraph/sourcegraph/pull/34783

This PR simply adds a header component to the page. The header layout looks slightly different from the original designs in order to simplify implementation and will be improved after the insight standalone page v1 (I think we will handle this right after the 3.40 cc @AlicjaSuska @Joelkw)  

<img width="1275" alt="Screenshot 2022-05-04 at 16 12 16" src="https://user-images.githubusercontent.com/18492575/166644476-f202c379-fd2c-4b14-a177-48eccbb0f058.png">

<img width="949" alt="Screenshot 2022-05-04 at 16 12 31" src="https://user-images.githubusercontent.com/18492575/166644534-fdeafb32-f7f9-4371-b042-9129ef2e0ed4.png">


## Test plan
- Make sure that all actions (copy link, edit, delete insight) over insight work properly 
- Make sure that the header layout looks on all viewports

## App preview:

- [Web](https://sg-web-vk-add-insight-standalone-header.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-zchouxmkab.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
